src/main/java/io/netty/incubator/codec/quic/QuicCodecBuilder.java

```diff
@@ -90,6 +91,14 @@ public final B grease(boolean enable) {
         return self();
     }
 
+    /**
+     * Enables logging of secrets.
+     */
+    public final B logKeys() {
```

Comment:
This is not needed... We should just use a ChannelOption on the server side as well. This should just work.

Reply:
From a quick inspection, Quiche API is organized in a way that this is in fact necessary. This flag is  set on a `Config` struct and its passes into `tls_ctx` [here](https://github.com/cloudflare/quiche/blob/e18fa69865bbc532b92064fdd245de6165417910/src/lib.rs#L584-L586) and it sets a `keylog` pointer  into `SSL_CTX_*` [here](https://github.com/cloudflare/quiche/blob/79fa08dc4b5dfe028a306cb913bbd3ca8448e663/src/tls.rs#L250-L254). But `keylog` callback itself uses property from the  connection which would be set only in this `set_keylog_path` function [here](https://github.com/cloudflare/quiche/blob/b1316fcf53f5065477d2c86e8707a82928260217/src/ffi.rs#L432-L451). So you need both.

Reply:
Thought I can try update native config object in the same place I pass `keylogPath` down to native but I'm not sure if having config as a mutable object is a good idea